core/trino-main/src/main/java/io/trino/server/protocol/spooling/SpoolingEnabledConfig.java

```diff
@@ -25,8 +25,8 @@ public boolean isEnabled()
         return enabled;
     }
 
-    @Config("experimental.protocol.spooling.enabled")
-    @ConfigDescription("Enable experimental spooling client protocol server-side support")
+    @Config("protocol.spooling.enabled")
```

Comment:
legacy config?

Reply:
Not needed. This wasn't announced yet so this isn't a breaking change.